# Probabilities from the normal distribution
Since each deal Amir worked on (both won and lost) was different, each was worth a different amount of money. These values are stored in the amount column of amir_deals and follow a normal distribution with a mean of 5000 dollars and a standard deviation of 2000 dollars. As part of his performance metrics, you want to calculate the probability of Amir closing a deal worth various amounts.

norm from scipy.stats is imported as well as pandas as pd. The DataFrame amir_deals is loaded.

In [3]:
from scipy.stats import norm
import pandas as pd

# Parameters of the distribution of Amir's sales
amir_mean = 5000
amir_std = 2000

What's the probability of Amir closing a deal worth less than $7500?

In [6]:
print(norm.cdf(7500, amir_mean, amir_std))

0.8943502263331446


What's the probability of Amir closing a deal worth more than $1000?

In [9]:
print(1 - norm.cdf(1000, amir_mean, amir_std))

0.9772498680518208


What's the probability of Amir closing a deal worth between $3000 and $7000?

In [10]:
print(norm.cdf(7000, amir_mean, amir_std) - norm.cdf(3000, amir_mean, amir_std))

0.6826894921370859


What amount will 25% of Amir's sales be less than?

In [19]:
print(norm.ppf(0.25, amir_mean, amir_std))

3651.0204996078364


Nifty normal distribution usage! You know that you can count on Amir 75% (1-0.25) of the time to make a sale worth at least $3651.02. This information could be useful in making company-wide sales projections.